src/coprocessor/dag/rpn_expr/types/mod.rs

```diff
@@ -56,4 +59,16 @@ impl<'a> RpnFnCallPayload<'a> {
     pub fn output_rows(&'a self) -> usize {
         self.output_rows
     }
+
+    /// Gets the length of implicit parameters
+    #[inline]
+    pub fn imp_params_len(&'a self) -> usize {
```

Comment:
imp may be confused with impl. I prefer to use implicit.

Also, it should be called as argument as `raw_args` above.

> A parameter is a variable in a method definition. When a method is called, the arguments are the data you pass into the method's parameters.

Reply:
What about `implicit_args_len`

Reply:
Looks good!